src/stories/src/components/expansion-panel/code/expansion-panel-card-recipe.ts

```diff
@@ -1,10 +1,10 @@
 export const ExpansionPanelCardRecipeCodeHtml = () => `
 <forge-card>
   <forge-expansion-panel>
-    <div role="button" tabindex="0" slot="header" style="display: flex; justify-content: space-between; align-items: center;">
+    <button slot="header" style="all: unset; display: flex; justify-content: space-between; align-items: center; width: 100%;">
```

Comment:
I see this example already had the styles set via an inline `style`, but could we split this out into a separate CSS snippet? I've been trying to fix these snippets as I go to remove inline styles to avoid it getting copied around.